src/util.rs

```diff
@@ -8,14 +8,14 @@ use std::time;
 
 use csv;
 use docopt::Docopt;
-use num_cpus;
+use num_cpus as ncpus;
 use serde::de::{Deserializer, Deserialize, DeserializeOwned, Error};
 
-use CliResult;
-use config::{Config, Delimiter};
+use crate::CliResult;
+use crate::config::{Config, Delimiter};
 
 pub fn num_cpus() -> usize {
-    num_cpus::get()
+    ncpus::get()
```

Comment:
Why not just use `num_cpus` here? The renaming doesn't seem worth it.

Reply:
There was a compiler error with 2018 over conflicting names.  Tbh I didn't look if there was a better way of resolving it so I just renamed the import.  I'll look to see if there's a better way.